source/AI.cpp

```diff
@@ -4171,6 +4171,9 @@ void AI::MovePlayer(Ship &ship, Command &activeCommands)
 			}
 			message += " in the system you are jumping to.";
 			Messages::Add(message, Messages::Importance::Info);
+
+			if(Preferences::Has("Play ding on destination"))
+				Audio::Play(Audio::Get("fail"));
```

Comment:
I'm not sure if the failure sound fits here.

Reply:
I think it works pretty well in the game.

Reply:
> I think it works pretty well in the game.

The sound plays if there's something important the player has to care about, and well, having to land on a planet for a mission is pretty important, so the only thing we might need is to change the name from "fail" to something more general in another PR.